homeassistant/components/lifx/light.py

```diff
@@ -424,6 +433,26 @@ def unregister(self, bulb):
             entity.registered = False
             entity.async_write_ha_state()
 
+    async def entity_registry_updated(self, event):
+        """Handle entity registry updated."""
+        if event.data["action"] == "remove":
+            await self.remove_empty_devices()
+
+    async def remove_empty_devices(self):
+        """Remove devices with no entities."""
+        entity_reg = await er.async_get_registry(self.hass)
```

Comment:
We prefer using `er.async_get`. The other is legacy.

Reply:
Got it, thank you.